src/example/Example.js

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
 import App from './App';
 
+const root = document.createElement('div');
```

Comment:
This is a good idea feel free to PR this to https://github.com/nkbt/react-component-template